ui/app/templates/components/job-page/system.hbs

```diff
@@ -11,12 +11,16 @@
     {{/each}}
   {{/if}}
 
-  <JobPage::Parts::JobClientStatusSummary
-    @job={{this.job}}
-    @jobClientStatus={{this.jobClientStatus}}
-    @gotoClients={{this.gotoClients}} />
+  {{#if (can "read client")}}
+    <JobPage::Parts::JobClientStatusSummary
+      @job={{this.job}}
+      @jobClientStatus={{this.jobClientStatus}}
+      @gotoClients={{this.gotoClients}} />
+  {{else}}
+    <JobPage::Parts::JobClientStatusSummary @job={{this.job}} @forceCollapsed="true" />
+  {{/if}}
 
-  <JobPage::Parts::Summary @job={{this.job}} @forceCollapsed="true" />
+  <JobPage::Parts::Summary @job={{this.job}} @forceCollapsed={{if (can "read client") true false}} />
```

Comment:
Should this read `if (can "read client")`?

Reply:
Or just `{{can "read client"}}`?